cli/config/configuration-fc.toml.in

```diff
@@ -289,3 +289,9 @@ disable_guest_seccomp=@DEFDISABLEGUESTSECCOMP@
 # If you are using docker, `disable_new_netns` only works with `docker run --net=none`
 # (default: false)
 #disable_new_netns = true
+
+# If enabled, the runtime will enable experimental features.
+# Experimental features are not stable enough for production usage so far,
```

Comment:
Suggested rewrite 294-295 - I am not certain this suggestion retains the original meaning so adjust as necessary: 
At this time, experimental features are not stable enough for production. These features will break compatibility, and are prepared for a big version bump.

Reply:
addressed in #1227